src/platform/datapath_raw.h

```diff
@@ -474,11 +437,52 @@ typedef struct TCP_HEADER {
 #define TH_CWR 0x80
 
 #define IPV4_VERSION 4
-#define IPV6_VERSION 6
 #define IPV4_VERSION_BYTE (IPV4_VERSION << 4)
-#define IPV4_DEFAULT_VERHLEN ((IPV4_VERSION_BYTE) | (sizeof(IPV4_HEADER) / sizeof(uint32_t)))
 
 #define IP_DEFAULT_HOP_LIMIT 128
 
+#ifndef _KERNEL_MODE
```

Comment:
why are these defs kernel-specific?

Reply:
These types seem to be defined somewhere for kernel mode